### DISCUSSION TITLE
Error wit login y Email

### DISCUSSION THREAD

**Discussion by eshams95** [WalletConnect/walletconnect-monorepo#4388] [Developer Support]
Hi
I have a series problems of using email login
1- When I enable login by any wallet (MetaMask, ...) after login wallet_id changed to email.
2- When I login by email after refresh the page it have delay to check the login and once disconnected and after 15 seconds automatically connect to email wallet.

What should I do to solve this bugs?

**Comment by glitch-txs** [comment]
hi, are you able to reproduce here: https://lab.web3modal.com/library/wagmi-email/

Could you share the steps to reproduce the issue?

**Reply by eshams95** [reply]
I use @web3modal/ethers package in my project and my project is react and typescript
